typescript/phoenix-sdk/config.json

```diff
@@ -32,7 +32,7 @@
       },
       {
         "name": "Wrapped SOL",
-        "symbol": "wSOL",
+        "symbol": "SOL",
```

Comment:
Should probably keep wSOL since it's wrapped? @eugene-chen what do you think

If this was to fix the issue in tests/watch.ts you can just make it so that it searches for "wSOL/USDC" rather than "SOL/USDC"

Reply:
agreed. this should be wSOL to be consistent with the master tokenlist (https://raw.githubusercontent.com/solana-labs/token-list/main/src/tokens/solana.tokenlist.json). 

if we have some sort of search we should special-case it so if you search SOL or wSOL this comes up

Reply:
I don't care if we change it for local or for devnet because those are our own tokens.